go/libraries/doltcore/sqle/enginetest/dolt_queries.go

```diff
@@ -3568,3 +3568,370 @@ var CommitDiffSystemTableScriptTests = []queries.ScriptTest{
 		},
 	},
 }
+
+var verifyConstraintsSetupScript = []string{
```

Comment:
i'm not sure this works for all of your tests, but using this as the harness setup -> `harness.Setup(verifyConstraintsSetu)`, and having the rest as regular `queries.QueryTest` or `queries.WriteQueryTest` is generally faster